internal/transport/http2_client.go

```diff
@@ -1238,7 +1238,7 @@ func (t *http2Client) handleRSTStream(f *http2.RSTStreamFrame) {
 	if statusCode == codes.Canceled {
 		if d, ok := s.ctx.Deadline(); ok && !d.After(time.Now()) {
 			// Our deadline was already exceeded, and that was likely the cause
-			// of this cancellation.  Alter the status code accordingly.
+			// of this cancellation. Alter the status code accordingly.
```

Comment:
Please revert.

Reply:
It's not a typo.  :)

"Two spaces after periods" was recommended when typewriters were used, because they were fixed-width, and it helped with readability:

https://graphicdesign.stackexchange.com/questions/2160/typesetting-two-spaces-after-the-period

I prefer continuing the tradition with source code since it isn't rendered in a variable-width font (also it's a habit), but I am fine with people wanting to use one space.  We shouldn't go around changing it from one way to the other, though, as it creates churn and is just a preference, not a typo.

Reply:
Thanks for sharing, wasn't aware of this.